templates/aes-agent.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if and .Values.agent.enabled .Values.enableAES }}
+{{- if not (semver .Values.image.tag | (semver "1.12.0").Compare) }}
```

Comment:
This would make it so that the agent is **only** installed with version `1.12.0` right?

It seems like we would want this to be a install the agent when the version is greater than or equal to `1.12.0`